dwave/optimization/states.pyx

```diff
@@ -200,7 +200,9 @@ cdef class States:
             self._states[i].resize(model.num_nodes())
             model._graph.initialize_state(self._states[i])
 
-    def into_file(self, file):
+    def into_file(self, file, *,
+                  version = None,
```

Comment:
missing types?

Reply:
The typing in Cython files doesn't get propagated, it goes in the `.pyi` file instead. Annoyingly.